src/utils.ts

```diff
@@ -106,3 +108,17 @@ export const getZoneLabel = (zone?: any) => {
 
   return label
 }
+
+export const migrateConfiguration = (configuration: any, version: number) => {
+  let updatedConfiguration = { ...configuration }
+
+  if (version < saveVersion) {
+    for (let i = version; i < saveVersion; i += 1) {
```

Comment:
You can drop the `if` because if `version >= saveVersion`, then the for loop won't run at all.